group_vars/kube-workers

```diff
@@ -0,0 +1 @@
+kube_role: worker
```

Comment:
I think this file can be removed in favour of a different approach (see my other comments).